src/main/resources/data/minecolonies/crafterrecipes/blacksmith/platearmorhelmet.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type" : "recipe",
+  "crafter": "blacksmith",
+  "inputs": [
+    {
+      "item" : "minecraft:iron_ingot",
+      "count": 4
+    },
+    {
+      "item" : "minecraft:leather",
+      "count": 1
+    },
+    {
+      "item" : "minecraft:coal",
+      "count": 1
+    }
+  ],
+  "result": "minecolonies:plate_armor_helmet",
+  "count": 1,
+  "min-building-level" : 4,
```

Comment:
Given that the research now enforces this, it's probably not necessary in the recipe definition, but it's also not critical.

Reply:
If the player has multiple blacksmith including some < lvl 4, this will prevent him to build the plate armors in the lower lvl blacksmith I guess

Reply:
Fair point.